tests/unit/testsuite/Bolt/Boltpay/OrderHelper.php

```diff
@@ -145,7 +145,11 @@ public static function deleteDummyOrderByIncrementId($incrementId)
      */
     public static function deleteDummyOrder(Mage_Sales_Model_Order $order)
     {
-        self::deleteDummyOrderByIncrementId($order->getIncrementId());
-
+        if (!empty($order)) {
+            $id = $order->getIncrementId();
+            if (!empty($id)) {
```

Comment:
When is it in our test that we have an order without an `incrementId`?

Here, we appear to be simply hiding errors in our test and should instead fix the erroneous testing conditions instead of allowing them to persist, and also, create a more robust and comprehensive `deleteDummyOrder` function.

Since we know that `$order` will be an instance of `Mage_Sales_Model_Order`  we can call `Mage_Sales_Model_Order::delete()`.  This is more robust and comprehensive as it doesn't require the order to have an increment id, and at the same time cleans up the orders that do not have this instead of leaving them unhandled as in this code. 

Also, since